src/main/java/mate/academy/springboot/datajpa/controller/CategoryController.java

```diff
@@ -0,0 +1,60 @@
+package mate.academy.springboot.datajpa.controller;
+
+import mate.academy.springboot.datajpa.dto.request.CategoryRequestDto;
+import mate.academy.springboot.datajpa.dto.response.CategoryResponseDto;
+import mate.academy.springboot.datajpa.model.Category;
+import mate.academy.springboot.datajpa.service.CategoryService;
+import mate.academy.springboot.datajpa.service.mapper.RequestDtoMapper;
+import mate.academy.springboot.datajpa.service.mapper.ResponseDtoMapper;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/categories")
+public class CategoryController {
+    private final CategoryService categoryService;
+    private final RequestDtoMapper<CategoryRequestDto, Category> requestDtoMapper;
+    private final ResponseDtoMapper<CategoryResponseDto, Category> responseDtoMapper;
+
+    public CategoryController(
+            CategoryService categoryService,
+            RequestDtoMapper<CategoryRequestDto, Category> requestDtoMapper,
+            ResponseDtoMapper<CategoryResponseDto, Category> responseDtoMapper
+    ) {
+        this.categoryService = categoryService;
+        this.requestDtoMapper = requestDtoMapper;
+        this.responseDtoMapper = responseDtoMapper;
+    }
```

Comment:
You have lombok in your dependencies, so you can replace constructor with an annotation

Reply:
fixed